static/sass/styles-v1.scss

```diff
@@ -12,6 +12,13 @@
 // vendor
 @import 'vendors/global-nav/build';
 
+// patterns
```

Comment:
We don't want to couple old CSS and v1 CSS - the last step in this migration will be to nuke all old styles without having to worry about dependencies.